RoboJackets/robocup-software
# Update package list for Arch/Manjaro

## Description
Update packages.txt for use with Arch Linux (and Manjaro).

BTW I use ~Arch~ Manjaro

## Associated Issue
Issue #1086, among others.

## Design Documents
N/A

## Steps to test
### Test Case 1
1. Use a fresh copy of arch/Manjaro
2. Clone this repository
3. Run util/arch-setup
4. `make run-sim`

Expected result: Soccer runs out-of-the-box
